packages/documentation/src/stories/components/subnavigation/subnavigation.stories.ts

```diff
@@ -0,0 +1,94 @@
+import { Meta, StoryObj, Args, StoryContext } from '@storybook/web-components';
+import { BADGE } from '../../../../.storybook/constants';
+import { html } from 'lit';
+
+const meta: Meta = {
+  title: 'Components/Subnavigation',
+  parameters: {
+    badges: [BADGE.WEB_COMPONENT_CANDIDATE],
+  },
+  render: render,
+  argTypes: {
+    coloredBackground: {
+      name: 'Colored Background',
+      control: {
+        type: 'boolean',
+      },
+    },
+    alternate: {
+      name: 'Alternate',
+      control: {
+        type: 'boolean',
+      },
+    },
+    badges: {
+      name: 'Badges',
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+};
+
+export default meta;
+
+function render(args: Args, context: StoryContext) {
+  return html`
+    <div
+      class="subnavigation${args.alternate
+        ? ' subnavigation-alternate'
+        : ''}${args.coloredBackground ? ' bg-petrol' : ''}"
+    >
+      <div class="container container-fluid-xs container-fluid-sm">
+        <ul class="subnavigation-list">
+          <li class="subnavigation-item">
+            <a href="/?path=/docs/components-subnavigation--docs" class="subnavigation-link active">
+              Navitem active
+              ${args.badge
+                ? html`
+                    <span class="badge bg-active rounded-pill">19</span>
```

Comment:
This classes aren't necessary: 
```suggestion
                    <span class="badge">19</span>
```